packages/users-module/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@platzily/users-module",
+  "version": "1.0.0",
+  "description": "User services",
+  "author": "Jorge Arias Argüelles <jorgeariasarguelles@gmail.com>",
```

Comment:
The author should be the team owner of the service.

Reply:
Author property was updated by @gh0stl1m recommendation. ✅